### PR TITLE
README.md changes

### PR DESCRIPTION
**Pull request by AnthonyBSong**
Made changes to the README.md to make directions on installation and cloning the repository more clear for Hackathon participants.

### CODE REVIEW

File: README.md
Comment (YoruCathy) [inline README.md:1]: Link to tutorial?

File: README.md [hunk 1]
```diff
@@ -3,30 +3,46 @@
 # Here is the code for RCareWorld V1.0.0 🦾
 - Check the website https://emprise.cs.cornell.edu/rcareworld/
 
-We support Windows, Linux (Tested on Ubuntu), and MacOS. You need to make sure git and git-lfs is installed on your system. You should use powershell for windows, 
-and bash for Linux and MacOS.
+We support Windows, Linux (Tested on Ubuntu), and MacOS. You need to make sure git and git-lfs are installed on your system. You should use Git Bash if you have it (else Powershell) for windows,
```
Comment (YoruCathy) [inline README.md:6]: Link to tutorial?

File: README.md [hunk 2]
```diff
@@ -3,30 +3,46 @@
...
 
-We support Windows, Linux (Tested on Ubuntu), and MacOS. You need to make sure git and git-lfs is installed on your system. You should use powershell for windows, 
-and bash for Linux and MacOS.
+We support Windows, Linux (Tested on Ubuntu), and MacOS. You need to make sure git and git-lfs are installed on your system. You should use Git Bash if you have it (else Powershell) for windows, 
+and use bash for Linux and MacOS.
+
 # Clone this repo
-If you have ssh:
+
+Open up Terminal, Git Bash, or Powershell on your computer.
```
Comment (YoruCathy) [inline README.md:11]: Depends on what system you are using

File: README.md [hunk 3]
```diff
@@ -3,30 +3,46 @@
...
+
+If you already have anaconda installed our your system, skip installing `install_conda`. 
+
 ## Windows
 Run `install/Windows/install_conda.bat`, and then run `install/Windows/install_pyrcareworld.bat`.
 
 Example commands:
 `./install/Windows/install_pyrcareworld.bat`
 
+Note on Windows you may need to install Microsoft Visual C++ 14.0 or greater using Visual Studio Installer.
```
Comment (YoruCathy) [inline README.md:44]: Link to tutorial

File: README.md.save
Comment (YoruCathy) [inline README.md.save:1]: Remove this .save file

**Review by YoruCathy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AnthonyBSong) [reply README.md:6]: Fixed by adding information on how to install Git and Git Large File Storage

File: README.md (hunk 3 above)
Comment (AnthonyBSong) [reply README.md:67]: Added youtube video on how to do it.

File: README.md.save
Comment (AnthonyBSong) [reply README.md.save:1]: Removed the .save file

**Pull request merged**